wesnoth-units.po

```diff
@@ -6311,18 +6309,22 @@ msgid ""
 "felt the Jinn were watching over them, driving off malign forces of the "
 "night."
 msgstr ""
+"Džinové jsou tajemní duchové, o nichž není známo, že by měli nějaký vztah k "
+"duším zemřelých. Často je lze spatřit z dálky, jak se vznášejí po pouštních "
```

Comment:
Asi bych dal pro význam distance [přednost spřežce](http://nase-rec.ujc.cas.cz/archiv.php?art=3567), ale berte to jen jako návrh.
```suggestion
"duším zemřelých. Často je lze spatřit zdálky, jak se vznášejí po pouštních "
```